src/src/com/microsoft/aad/adal/TokenCacheItem.java

```diff
@@ -57,6 +57,8 @@
     private boolean mIsMultiResourceRefreshToken;
 
     private String mTenantId;
+    
+    private String mFamilyClientId;
```

Comment:
The reason I'm asking that this class implements Serializable, but I'm not sure if it actually ever used to serialize this object.